src/main/mulmo/handler_generator.ts

```
@@ -94,6 +95,11 @@
   }
 };
 
+const beatAudioPath = (context: MulmoStudioContext, beat) => {
```

ESLint

[error] 'MulmoStudioContext' is not defined.

(no-undef)